backend/curation/api/curation-api.yml

```diff
@@ -1313,15 +1341,47 @@ components:
       type: string
       nullable: true
       example: "10.1093/ajae/aaq063"
+    donor_id:
+      description: Free-text that identifies a unique individual that data were derived from.
+      type: array
+      items:
+        type: string
+      nullable: true
+    embeddings:
+      description: List of keys for embeddings in the dataset
+      type: array
+      items:
+        type: string
+      nullable: true
     explorer_url:
       description: |
         The URL at which a given Dataset may be explored using CELLxGENE Explorer.
       nullable: true
       type: string
-    donor_id:
-      description: Free-text that identifies a unique individual that data were derived from.
+    feature_biotype:
+      description: List of feature biotypes in dataset. Must be
```

Comment:
typo?

Reply:
The diff is showing up weirdly, but the donor_id isn't being deleted. I just moved it above 'explorer_url' to maintain alphabetical ordering

Reply:
> List of feature biotypes in dataset. Must be

Why does this line just end?